src/styles/main.scss

```diff
@@ -1,3 +1,180 @@
-body {
+* {
+  box-sizing: border-box;
+}
+
+h2 {
+  margin: 0;
+  vertical-align: baseline;
+  font-size: 100%;
+}
+
+p {
+  margin: 0;
+  vertical-align: baseline;
+  font-size: 100%;
+}
+
+img {
+  margin: 0;
+  vertical-align: baseline;
+  font-size: 100%;
+}
+
+button {
   margin: 0;
+  vertical-align: baseline;
+  font-size: 100%;
+}
+
+html {
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+:root {
+  --main-accent: #060b35;
+  --blue-accent: #00acdc;
+  --white-accent: #fff;
+  --secondary-accent: #616070;
+  --colomn-count: 1;
+}
+
+.card {
+  padding-left: 16px;
+  padding-right: 16px;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  background-color: var(--white-accent);
+
+  &__goods {
+    width: 160px;
+    height: 134px;
+    margin: 32px auto 0;
+    display: block;
+  }
+
+  &__info {
+    color: var(--main-accent);
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+    width: 166px;
+
+    &:hover {
+      color: #34568b;
+      // transition: transform 300ms;
+    }
+  }
+
+  &__code {
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: var(--secondary-accent);
+
+    padding-top: 4px;
+    padding-bottom: 16px;
+    margin: 0;
+  }
+
+  &-chosen {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    align-items: flex-end;
+  }
+
+  &-reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: var(--main-accent);
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 0 16px;
+    align-items: center;
+  }
+
+  &__price {
+    color: var(--secondary-accent);
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__sum {
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 18px;
+    color: var(--main-accent);
+  }
+
+  &__button {
+    display: flex;
+    color: var(--white-accent);
+    height: 40px;
+    width: 100%;
+    border-radius: 5px;
+    background-color: var(--blue-accent);
+    font-size: 14px;
+    margin-bottom: 16px;
+    justify-content: center;
+    align-items: center;
+
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    border: 1px solid transparent;
+    text-decoration: none;
+
+    &:hover {
+      border-color: var(--blue-accent);
+      background-color: #fff;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)